### PR TITLE
OCPQE-21011 update more specific filter

### PR DESCRIPTION
**Pull request by dis016**
[OCPQE-21011](https://issues.redhat.com//browse/OCPQE-21011) as per discussion in https://redhat-internal.slack.com/archives/C05EV34PSKF/p1713924512277759?thread_ts=1712545587.556329&cid=C05EV34PSKF updating the select query.
Follow up on https://github.com/openshift/release/pull/51049 

/cc @jianlinliu @jiajliu @JianLi-RH @shellyyang1989

### CODE REVIEW

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-4.16-upgrade-from-stable-4.15-azure-ipi-ovn-ipsec-f14

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-4.16-upgrade-from-stable-4.11-azure-ipi-ovn-ipsec-f30

File: ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh [hunk 1]
```diff
@@ -623,7 +623,7 @@ function upgrade() {
     set_channel $TARGET_VERSION
     local retry=3 conditional_updates
     while (( retry > 0 )); do
-        conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]? | select (.conditions[].status!="True")| .release.version' | xargs)
+        conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]? | select((.conditions[].type == "Recommended") and (.conditions[].status != "True")) | .release.version' | xargs)
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh:626]: a minor suggestion:
```suggestion
        unrecommened_conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]? | select((.conditions[].type == "Recommended") and (.conditions[].status != "True")) | .release.version' | xargs)
```

Comment (dis016) [reply ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh]: updated in new commit

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-4.16-upgrade-from-stable-4.11-azure-ipi-ovn-ipsec-f30

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.15-azure-ipi-fips-f28

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.15-azure-ipi-fips-f28

**Comment by jianlinliu** [comment]
btw, pls squash multiple commit into one.

**Comment by dis016** [comment]
success run : https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/51274/rehearse-51274-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.15-azure-ipi-fips-f28/1783376357376397312

**Pull request merged**